### PR TITLE
Renaming settings to Properties

### PR DESCRIPTION
**Pull request by kirankumarkolli**
Renaming <T>Settings to <T>Properties

### CODE REVIEW

File: Microsoft.Azure.Cosmos/src/ChangeFeedProcessor/FeedProcessing/FeedProcessorCore.cs [hunk 1]
```diff
@@ -19,7 +19,7 @@ namespace Microsoft.Azure.Cosmos.ChangeFeed.FeedProcessing
 
     internal sealed class FeedProcessorCore<T> : FeedProcessor
     {
-        private readonly ProcessorSettings settings;
+        private readonly ProcessorOptions options;
```
Comment (simplynaveen20) [inline Microsoft.Azure.Cosmos/src/ChangeFeedProcessor/FeedProcessing/FeedProcessorCore.cs]: Any reason we haven't use property in changefeed code ?

File: Microsoft.Azure.Cosmos/src/Resource/Conflict/CosmosConflicts.cs [hunk 2]
```diff
@@ -88,7 +88,7 @@ public abstract class CosmosConflicts
         /// FeedIterator<CosmosConflictSettings> conflictIterator = await cosmosContainer.Conflicts.GetConflictsIterator();
```
Comment (simplynaveen20) [inline Microsoft.Azure.Cosmos/src/Resource/Conflict/CosmosConflicts.cs]: CosmosConflictSettings  -> CosmosConflictProperties .

File: Microsoft.Azure.Cosmos/src/DocumentClient.cs [hunk 3]
```diff
@@ -6538,7 +6538,7 @@ private async Task InitializeGatewayConfigurationReaderAsync()
             this.accountServiceConfiguration = new CosmosAccountServiceConfiguration(accountReader.InitializeReaderAsync);
 
             await this.accountServiceConfiguration.InitializeAsync();
-            CosmosAccountSettings accountSettings = this.accountServiceConfiguration.AccountSettings;
+            CosmosAccountProperties accountSettings = this.accountServiceConfiguration.AccountProperties;
```
Comment (simplynaveen20) [inline Microsoft.Azure.Cosmos/src/DocumentClient.cs]: accountSettings  -> accountProperties

File: Microsoft.Azure.Cosmos/src/ChangeFeedProcessor/LeaseManagement/DocumentServiceLeaseManagerCosmos.cs [hunk 4]
```diff
@@ -17,13 +17,13 @@ internal sealed class DocumentServiceLeaseManagerCosmos : DocumentServiceLeaseMa
...
         private readonly DocumentServiceLeaseUpdater leaseUpdater;
-        private readonly DocumentServiceLeaseStoreManagerSettings settings;
+        private readonly DocumentServiceLeaseStoreManagerOptions settings;
         private readonly RequestOptionsFactory requestOptionsFactory;
 
         public DocumentServiceLeaseManagerCosmos(
             CosmosContainer leaseContainer,
             DocumentServiceLeaseUpdater leaseUpdater,
-            DocumentServiceLeaseStoreManagerSettings settings,
+            DocumentServiceLeaseStoreManagerOptions settings,
```
Comment (j82w) [inline Microsoft.Azure.Cosmos/src/ChangeFeedProcessor/LeaseManagement/DocumentServiceLeaseManagerCosmos.cs]:
```suggestion
            DocumentServiceLeaseStoreManagerOptions options,
```

File: Microsoft.Azure.Cosmos/src/ChangeFeedProcessor/LeaseManagement/DocumentServiceLeaseStoreManagerBuilder.cs [hunk 5]
```diff
@@ -14,7 +14,7 @@ namespace Microsoft.Azure.Cosmos.ChangeFeed.LeaseManagement
     /// </summary>
     internal class DocumentServiceLeaseStoreManagerBuilder
     {
-        private DocumentServiceLeaseStoreManagerSettings settings = new DocumentServiceLeaseStoreManagerSettings();
+        private DocumentServiceLeaseStoreManagerOptions settings = new DocumentServiceLeaseStoreManagerOptions();
```
Comment (j82w) [inline Microsoft.Azure.Cosmos/src/ChangeFeedProcessor/LeaseManagement/DocumentServiceLeaseStoreManagerBuilder.cs]: Please update all the parameter names.

File: Microsoft.Azure.Cosmos/src/ChangeFeedProcessor/FeedProcessing/FeedProcessorCore.cs (hunk 1 above)
Comment (kirankumarkolli) [reply Microsoft.Azure.Cosmos/src/ChangeFeedProcessor/FeedProcessing/FeedProcessorCore.cs]: Options is for choice. Properties for data-contracts.

**Pull request merged**